tensorflow/core/kernels/cwise_op_conj.cc

```diff
@@ -20,6 +20,7 @@ namespace tensorflow {
 REGISTER2(UnaryOp, CPU, "Conj", functor::conj, complex64, complex128);
 #if GOOGLE_CUDA
 REGISTER_KERNEL_BUILDER(Name("Conj").Device(DEVICE_GPU),
-                        UnaryOp<GPUDevice, functor::conj<complex64>>);
+                        UnaryOp<GPUDevice, functor::conj<complex64>>,
+                        functor::conj<complex128>>);
```

Comment:
I don't think this is syntactically correct -- let me konw when you've verified these changes.